processor/memorylimiterprocessor/memorylimiter.go

```diff
@@ -20,9 +20,9 @@ import (
 	"fmt"
 	"runtime"
 	"sync"
+	"sync/atomic"
```

Comment:
Shouldn't this change the processor's go.mod file as well to remove the direct dependency?

Reply:
Done